### PR TITLE
Docs: add message that storybook creates node_modules in pnp

### LINKED ISSUE #25987: [Documentation]:  Added a comment that yarn pnp mode is supported, but node_modules are created when storybook is run.
### Describe the problem

'yarn storybook' creates node_modules/.cache in Yarn 2 PnP project

[Discussion link](https://github.com/storybookjs/storybook/discussions/25986#discussioncomment-8432146)

So, I explained 'yarn storybook' creates node_modules/.cache in Yarn 2 PnP project install.md files

### PR DESCRIPTION
**Pull request by turtle601**
Closes #25987

## What I did

'yarn storybook' creates node_modules/.cache in Yarn 2 PnP project

I think the development environment can cause confusion for those developing in pnp mode.

so, i add 'yarn storybook' creates node_modules/.cache in Yarn 2 PnP project in get-started/install.md files

#### The changes in this PR are covered in the following automated tests:
- [ ] stories
- [ ] unit tests
- [ ] integration tests
- [ ] end-to-end tests

#### Manual testing

_This section is mandatory for all contributions. If you believe no manual test is necessary, please state so explicitly. Thanks!_

### Documentation

- [x] Add or update documentation reflecting your changes
- [ ] If you are deprecating/removing a feature, make sure to update
      [MIGRATION.MD](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md)

## Checklist for Maintainers

- [ ] When this PR is ready for testing, make sure to add `ci:normal`, `ci:merged` or `ci:daily` GH label to it to run a specific set of sandboxes. The particular set of sandboxes can be found in `code/lib/cli/src/sandbox-templates.ts`
- [ ] Make sure this PR contains **one** of the labels below:
   <details>
     <summary>Available labels</summary>

     - `bug`: Internal changes that fixes incorrect behavior.
     - `maintenance`: User-facing maintenance tasks.
     - `dependencies`: Upgrading (sometimes downgrading) dependencies.
     - `build`: Internal-facing build tooling & test updates. Will not show up in release changelog.
     - `cleanup`: Minor cleanup style change. Will not show up in release changelog.
     - `documentation`: Documentation **only** changes. Will not show up in release changelog.
     - `feature request`: Introducing a new feature.
     - `BREAKING CHANGE`: Changes that break compatibility in some way with current major version.
     - `other`: Changes that don't fit in the above categories.
   
   </details>

### 🦋 Canary release

This PR does not have a canary release associated. You can request a canary release of this pull request by mentioning the `@storybookjs/core` team here.

_core team members can create a canary release [here](https://github.com/storybookjs/storybook/actions/workflows/canary-release-pr.yml) or locally with `gh workflow run --repo storybookjs/storybook canary-release-pr.yml --field pr=<PR_NUMBER>`_

### CODE REVIEW

**Comment by turtle601** [comment]
@jonniebigodes I'm really looking forward to your feedback on that pr

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@turtle601 appreciate you taking the time to put together this pull request and helping us improve our documentation by adding this small but rather important piece of information that is affecting you and can affect other users as well. I left a small item for you to look into when you have a moment.

Let me know of any questions you may have so that I can unblock you and get this merged.

Looking forward to hearing from you.

Have a fantastic day.

Stay safe

File: docs/get-started/install.md [hunk 1]
```diff
@@ -74,6 +74,13 @@ Storybook collects completely anonymous data to help us improve user experience.
 
 </Callout>
 
+<Callout variant="info">
+
+If your repo is in PnP (Plug And Play) mode, node_modules may be installed when run.
+because Storybook's cache directory is located in the node_modules folder, since it is normally not checked in via version control.
+
+</Callout>
```
[the remark this answers is not in the document]
Comment (turtle601) [reply docs/get-started/install.md]: @jonniebigodes Thank you so much for your feedback. :)

I saw your feedback and reflected it exactly.

I would appreciate it if you could check my PR one more time.

**Review by jonniebigodes — APPROVED** [review]

**Comment by jonniebigodes** [comment]
@turtle601, no need to thank you whatsoever, glad to help you! I appreciate you addressing the feedback so quickly. I've checked, and all is good on my end. I'm going to merge this once the checklist clears. 

Hope you have a fantastic day.

**Pull request merged**